### ISSUE TITLE
Can't select typeahead on mobile

### ISSUE THREAD

**Issue by jbrozena22** [sergeyt/meteor-typeahead#44]
Hello,

I have had success using your package for the web. The package also works on mobile safari, which is great. However, when I try using it with iOS through phonegap, the typeahead suggestion is not selectable. When I press the typeahead selection, rather than autocompleting the remainder of the phrase, the suggestion dropdown disappears, and the text is left wherever you stopped typing.

This is tested with: sergeyt:typeahead@0.10.5_7, Meteor 1.0, iOS 8.0 (both iOS simulator and on physical device)

To test this out, try running the following on the demo app provided with this project (assumes you have XCode and iOS Simulator installed):

meteor add-platform ios
meteor run ios

In the first NBA teams textbox enter Char
Charlotte Bobcats witll appear as a suggestion
Click on the suggestion

Expected response: textbox autocompletes with Charlotte Bobcats
Actual response: textbox remains at Char and the suggestion dropdown disappears

**Comment by cameroncruz** [comment]
I am having the same issue.  It seems that on iOS (using Cordova), when the keyboard is active, tapping anywhere outside the keyboard will not register in the app.  Instead, tapping outside the keyboard will first hide the keyboard and unfocus from the input element.  Any tap events after that will trigger normally.

With typeahead, the suggestion dropdown is closed if the user unfocuses from the input element.  Therefore, on iOS, the user is never in a situation where they can tap on a suggestion and have it autocomplete.  

Is it possible to prevent the typeahead dropdown from closing when the user unfocuses from the input element?  Or manually control when/how a typeahead input is opened/closed? 

This is not an issue on Android devices, at least in my experience.

**Comment by sergeyt** [comment]
@cameroncruz I think it is possible to reproduce the issue using [typeahead](https://github.com/twitter/typeahead.js) inside non-meteor Cordova environment, right? This package is wrapper around typeahead and such kind of issue should be fixed in typeahead code rather than wrapper. But if you find a workaround fix we could add it the package temporarily until the fix is included to typeahead code.

**Comment by cameroncruz** [comment]
@sergeyt Yes, after researching this further I found [this issue](https://github.com/twitter/typeahead.js/issues/792) and I guess it is in fact a conflict between typeahead and fastclick? 

The fix suggested in that issue was to comment out `this.dropdown.empty();` in onBlurred() like this:

`_onBlurred: function onBlurred() {                                                                         // 1522`
`                this.isActivated = false;                                                                              // 1523`
`                //this.dropdown.empty();                                                                                 // 1524`
`                this.dropdown.close();                                                                                 // 1525`
`            }, `

I haven't tested this for myself yet.  But you're right, this is not an issue with your package, and it seems that the people working on typeahead are already aware of the problem.

**Comment by sergeyt** [comment]
@cameroncruz thanks for research and info

**Comment by designbyadrian** [comment]
:+1: also have this issue.
